cas-management-webapp-support/src/main/java/org/jasig/cas/config/CasManagementWebAppConfiguration.java

```diff
@@ -43,6 +43,7 @@
  * @since 4.3.0
  */
 @Configuration("casManagementWebAppConfiguration")
+@Lazy(true)
```

Comment:
What's the idea behind this `@Lazy(true)` as all beans will be needed, won't they?